.gitignore

```diff
@@ -5,3 +5,6 @@ __pycache__
 .vscode
 uploads
 
+
+.space
+.vercel
```

Comment:
just add the platform we will keep, so probably render. you can remove folders of platforms we won't use